src/Cli/Microsoft.DotNet.Cli.Sln.Internal/FileManipulation/FileUtil.cs

```diff
@@ -1,28 +1,5 @@
-//
-// FileUtil.cs
-//
-// Author:
-//       Lluis Sanchez Gual <lluis@xamarin.com>
-//
-// Copyright (c) 2015 Xamarin, Inc (http://www.xamarin.com)
```

Comment:
In general, it is not ok to delete Xamarin and Novell copyrights like this (in dotnet/runtime at least).

Reply:
Done.